### PR TITLE
Make rule spring boot rule available for spring apps

### PR DESCRIPTION
**Pull request by showpune**
1)  Make rule spring boot rule azure target to azure spring apps
2) Update the doc link to azure spring apps

### CODE REVIEW

**Comment by agoncal** [comment]
@showpune looks good to me. But you should internationalize the documentation links removing the specific `en-us`. So instead of `https://learn.microsoft.com/en-us` consider just having `https://learn.microsoft.com`.

If you want the Azure Spring Apps target to appear in the Web Console, you should add it here:

https://github.com/windup/windup-web/blob/master/ui-pf4/src/main/webapp/src/layout/TransformationPaths.ts#L117

**Comment by showpune** [comment]
Thanks @agoncal  for reminder
1) The link is changed from https://learn.microsoft.com/en-us to https://learn.microsoft.com
2) the ts change is followed by PR https://github.com/windup/windup-web/pull/907

File: rules/rules-reviewed/azure/azure-file-system.windup.xml [hunk 1]
```diff
@@ -57,7 +58,7 @@
                         Any usage of the file system on the application will require reconfiguration or, in rare cases, architectural changes.
                         ]]>
                     </message>
-                    <link title="Relative path found in the application" href="https://docs.microsoft.com/azure/developer/java/migration/migrate-spring-boot-to-app-service#determine-whether-and-how-the-file-system-is-used"/>
+                    <link title="Relative path found in the application" href="https://learn.microsoft.com/en-us/azure/developer/java/migration/migrate-spring-cloud-to-azure-spring-apps?pivots=sc-standard-tier#determine-whether-and-how-the-file-system-is-used"/>
```
Comment (mrizzi) [inline rules/rules-reviewed/azure/azure-file-system.windup.xml:61]: @showpune Hi!
Sorry I can still see some `en-us` references in some URLs: do you mind changing all of the URLs to fulfill Antonio's request about internationalization?
Thanks.

File: rules/rules-reviewed/azure/azure-file-system.windup.xml [hunk 2]
```diff
@@ -80,7 +81,7 @@
                         Any usage of the file system on the application will require reconfiguration or, in rare cases, architectural changes.
                         ]]>
                     </message>
-                    <link title="Home path found in the application" href="https://docs.microsoft.com/azure/developer/java/migration/migrate-spring-boot-to-app-service#determine-whether-and-how-the-file-system-is-used"/>
+                    <link title="Home path found in the application" href="https://learn.microsoft.com/en-us/azure/developer/java/migration/migrate-spring-cloud-to-azure-spring-apps?pivots=sc-standard-tier#determine-whether-and-how-the-file-system-is-used"/>
```
Comment (mrizzi) [inline rules/rules-reviewed/azure/azure-file-system.windup.xml:84]: Same as above about `en-us` reference removal.

File: rules/rules-reviewed/azure/springboot/spring-boot-to-azure-database.windup.xml [hunk 3]
```diff
@@ -55,7 +56,7 @@
                         Checkout the different types of databases that are fully managed on Azure.
                         ]]>
                         </message>
-                    <link title="MongoDB connection string found in configuration file" href="https://docs.microsoft.com/azure/developer/java/migration/migrate-spring-boot-to-app-service#databases"/>
+                    <link title="MongoDB connection string found in configuration file" href="https://learn.microsoft.com/en-us/azure/developer/java/migration/migrate-spring-boot-to-azure-spring-apps#databases"/>
```
Comment (mrizzi) [inline rules/rules-reviewed/azure/springboot/spring-boot-to-azure-database.windup.xml:59]: Same as above about `en-us` reference removal.

**Review by mrizzi — COMMENTED** [review]
Reviewed changes checking them against previous comment from Antonio and added some further comments.

**Review by mrizzi — APPROVED** [review]
@showpune thanks a lot for your first contribution :clap:

**Pull request merged**